riot-os-posix-lwip/README.md

```diff
@@ -0,0 +1,95 @@
+# Posix sockets: TLS client/server example
+
+This example shows how to use wolfSSL TLS sockets over RIOT-OS POSIX sockets
+
+
+## License
+
+This example is distributed under the terms of GNU GPLv2.
+
+## SOCK vs. Socket
+
+This example is configured to use wolfSSL on RIOT-OS over POSIX sockets (LWIP).
+
+
+## Fast configuration (Between RIOT instances)
+
+### Prepare the bridge interface linking two tuntap
+
+```bash
+    ./../../dist/tools/tapsetup/tapsetup --create 2
+```
+
+## Testing
+
+### Prerequisite
+
+Clone RIOT-OS in a directory on your filesystem.
+
+### Run the server
+```bash
+$ make all RIOTBASE=/path/to/riot-os; PORT=tap1 make term
```

Comment:
Had issue on Ubuntu. Needed to install `sudo apt-get install gcc-multilib`.

Reply:
On the `PORT=tap1 make term` I needed to add sudo. Got:
```
RIOT TLS testing implementation
/home/davidgarske/GitHub/wolfssl-examples/riot-os-posix-lwip/bin/native/posix-tls.elf: ioctl TUNSETIFF: Operation not permitted
/home/davidgarske/GitHub/wolfssl-examples/riot-os-posix-lwip/bin/native/posix-tls.elf: probably the tap interface (tap1) does not exist or is already in use
make: *** [/home/davidgarske/GitHub/wolfssl-examples/riot-os-posix-lwip/../../RIOT/Makefile.include:749: term] Error 1
```
Otherwise `chmod /dev/net/tune` as you said.. Might be good to add note.